navigation.yaml

```diff
@@ -75,6 +75,8 @@ containers:
       path: /kubernetes/install
     - title: Kubernetes partners
       path: /kubernetes/partners
+    - title: Kubernetes features
```

Comment:
can you make these all just the final word, no Kubernetes?

Install, Managed, Partners, Features, etc?

Sorry

Reply:
Done